src/RecoveryQuestionController.js

```diff
@@ -82,7 +82,7 @@ function (Okta, FormController, FormType, FooterSignout, TextBox) {
     },
     initialize: function () {
       if (!this.settings.get('features.hideBackToSignInForReset')) {
-        this.addFooter(FooterSignout, {linkText: Okta.loc('goback', 'login'), linkClassName: ''});
```

Comment:
In this controller, `RecoveryQuestionController.js`, we are still going through the process of "authenticating". So I don't think this one needs to be reverted to "Sign out". This should be whatever was agreed with UX

Reply:
Well, it was `Sign out ` before this change anyway. And we agreed with UX that in those two cases the text can be either `Sign out` or `Back to Sign In`.

Reply:
got it. Will defer to @gowthamidommety-okta on this one, the rest of the PR looks good

Reply:
👍 changing it back to Sign out for now!

Reply:
Sorry, i did not realize this was src/RecoveryQuestionController.js. Agree with Mau, this should be Back To Sign In. src/PasswordResetController.js should stay saying Sign Out